prisma/schema.prisma

```diff
@@ -0,0 +1,62 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+
+enum Usertype {
+  USER
+  ADMIN
+}
+
+model User {
+  id          Int      @id @default(autoincrement())
+  email       String   @unique @map("email")
+  name        String   @map("name")
+  password    String   @map("password")
+  userType    Usertype @default(USER) @map("user_type")
+  dateCreated DateTime @default(now()) @map("date_created")
+  dateUpdated DateTime @updatedAt @map("date_updated")
+
+  userProfile Userprofile ?
+
+  toDo          Todo[]
+  userprofileId Int
+
+  @@map("User")
```

Comment:
@@map("user")

Reply:
Database table or column names are in small caps with snare-casing where necessary